easybuild/easyconfigs/b/Bash/Bash-4.3-GCC-4.9.2.eb

```diff
@@ -0,0 +1,31 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg/Computer Science and Communications Research Unit
+# Authors::   Valentin Plugaru <valentin.plugaru@gmail.com>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_05-06.html
+##
+
```

Comment:
Ok, fixed.

On Sat, Jan 17, 2015 at 12:37 AM, Kenneth Hoste <notifications@github.com>
wrote:

> In easybuild/easyconfigs/b/Bash/Bash-4.3-GCC-4.9.2.eb
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/1308#discussion-diff-23117450>
> :
>
> > @@ -0,0 +1,31 @@
> > +##
> > +# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
> > +#
> > +# Copyright:: Copyright 2012-2013 University of Luxembourg/Computer Science and Communications Research Unit
> > +# Authors::   Valentin Plugaru <valentin.plugaru@gmail.com>
> > +# License::   MIT/GPL
> > +# $Id$
> > +#
> > +# This work implements a part of the HPCBIOS project and is a component of the policy:
> > +# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_05-06.html
> > +##
> > +
>
> this is missing easyblock = 'ConfigureMake' too
>
> the build works with EasyBuild v1.16.1 (cfr. test report), but it won't
> with EasyBuild v2.0 (and Jenkins will tell you that with failing tests)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hpcugent/easybuild-easyconfigs/pull/1308/files#r23117450>
> .
>



-- 
- dr. Geert Jan BEX
- HPC analyst/consultant Hasselt University & University of Leuven
- email:                geertjan.bex@uhasselt.be
- homepage:         http://alpha.uhasselt.be/~gjb/
- PGP public key: http://alpha.uhasselt.be/~gjb/pgp.txt
---------------------------------------------------------